csv_template_ingester/template_csv_parser.py

```diff
@@ -548,8 +550,16 @@ def add_property(systm, property_value, names, units, column_index):
     if not property_value:
         return systm
 
-    property_value = listify(property_value)
-    prop.scalars = property_value
+    if isinstance(property_value, str) and re.match(range_pattern, property_value):
+        min_max = re.match(range_pattern, property_value).groupdict()
+        minimum = float(min_max["min"])
+        maximum = float(min_max["max"])
+        if minimum > maximum:
+            raise ValueError("Minimum ({}) cannot be greater than maximum ({})".format(minimum, maximum))
+        prop.scalars = Scalar(minimum=minimum, maximum=maximum)
```

Comment:
`listify` returns a list, so this scalar should be wrapped in a list as well.

Reply:
Good point, I'll return a list